Sources/Signal.swift

```diff
@@ -1459,37 +1442,33 @@ extension Signal {
 }
 
 extension Signal where Value: Equatable {
-	/// Forward only those values from `self` which are not duplicates of the
-	/// immedately preceding value. 
+	/// Forward only values from `self` that are not equal to its consecutive predecessor.
```

Comment:
I think _the immediately preceding value_ is clearer here. I'm not sure _consecutive predecessor_ makes sense.